examples/twentyseven-pt-stencil-solver/doc/results.dox

```diff
@@ -1,9 +1,123 @@
 <h1>Results</h1>
 
-The expected output of the relative error at K=10 should be
+The expected output at K=10 should be
 
 @code{.cpp}
-The average relative error is 1.87318e-15
+
+0.00679994 0.0139934 0.0268202 0.0486924 0.082948 0.132891 0.201945 0.294333 0.419256 0.62469
+0.0168018 0.0283773 0.0466095 0.0760775 0.121134 0.186034 0.27509 0.393188 0.547571 0.750224
+0.0345538 0.048906 0.0697503 0.102045 0.150438 0.219392 0.313283 0.436533 0.593739 0.788847
+0.0650736 0.0813831 0.103811 0.137521 0.18728 0.257596 0.352752 0.476773 0.633213 0.824361
+0.113021 0.13069 0.154094 0.188564 0.23895 0.309774 0.405243 0.529173 0.684736 0.873775
+0.182903 0.201361 0.225224 0.259998 0.310613 0.38161 0.477163 0.600991 0.756096 0.944096
+0.279102 0.297686 0.321454 0.3561 0.406635 0.477632 0.573268 0.697229 0.852432 1.0403
+0.405836 0.423664 0.446652 0.480674 0.530808 0.601661 0.697443 0.821873 0.977866 1.16669
+0.566998 0.582782 0.604089 0.636879 0.686217 0.756726 0.852682 0.977939 1.13558 1.32694
+0.765622 0.777394 0.795863 0.82666 0.874668 0.944475 1.04046 1.16675 1.32695 1.52323
+:
+0.0167651 0.0283072 0.0465118 0.0759595 0.121004 0.185899 0.274962 0.393077 0.547489 0.75018
+0.0291853 0.0463483 0.0689827 0.102026 0.150062 0.217457 0.308533 0.427951 0.581508 0.77596
+0.0491702 0.0708768 0.0972465 0.133535 0.184549 0.254761 0.348388 0.469474 0.621805 0.807902
+0.0814686 0.106594 0.135639 0.174097 0.226922 0.298608 0.393213 0.514273 0.664501 0.845005
+0.130712 0.15823 0.189002 0.228752 0.28255 0.354906 0.449762 0.57036 0.718928 0.896063
+0.201362 0.230216 0.261794 0.302066 0.356228 0.428829 0.523757 0.644108 0.791863 0.967311
+0.297681 0.326626 0.357976 0.397977 0.451965 0.524544 0.619611 0.740215 0.888206 1.06359
+0.423659 0.45105 0.48089 0.519702 0.572929 0.645238 0.74059 0.862101 1.01162 1.18894
+0.582779 0.606285 0.632968 0.669474 0.721228 0.792923 0.888647 1.01175 1.1644 1.34646
+0.777392 0.793673 0.81518 0.848059 0.897436 0.967948 1.06387 1.18904 1.34649 1.53734
+:
```

Comment:
is any idea why we use `:` to separate?

Reply:
I have no idea, I think that was done by a student assist.

Reply:
It was a student from the HPC course, I believe.